wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js

```diff
@@ -350,6 +350,44 @@
 	 */
 	Wicket.Ajax.Call = Wicket.Class.create();
 
+	Wicket.Ajax.Call._suspended = 0;
+
+	Wicket.Ajax.Call.currentNotify = undefined;
+
+	Wicket.Ajax.Call.suspend = function () {
+		if (typeof (Wicket.Ajax.Call.currentNotify) != "function") {
+			Wicket.Log.error("Can't suspend: no evaluation in process");
+			return;
```

Comment:
IMO this should either log error and return a no-op function or throw an exception instead of just logging.

Reply:
I'll go with throw